src/scripts/modal.js

```diff
@@ -0,0 +1,37 @@
+function modal() {
+  const heroContent = document.querySelector(".hero__content");
+  const modalContent = document.querySelector("#modal");
+  const checkOfferBtn = document.querySelector(".hero--cta");
+  const confrimModalBtn = modalContent.querySelector(".modal-btn--confirm");
```

Comment:
This variable has a typo and is not used anywhere. It can be deleted

Reply:
removed.